### PR TITLE
Fetchdata from api feature

### LINKED ISSUE #6: Create CI-CD Pipeline

### PR DESCRIPTION
**Pull request by shakyav**
Fetch data component is created, which gathers data from mock postman server 
https://bc2f9a92-a71f-4ad8-b93b-63ad65e50fa0.mock.pstmn.io/cluster_data
with the help of another package called "Axios" .

use the `<Fetchdata/> `tag whereever you want to add this component and add following code for import:
`import { Fetchdata } from '@app/fetchdata/fetchdata'`

It will look like the below image if the component is rendered inside Openshift component

image

### CODE REVIEW

**Comment by gagansk** [comment]
@shakyav , can you please rebase the code? I merged a PR and it has merge conflicts.

File: package.json [hunk 1]
```diff
@@ -82,6 +82,7 @@
     "@patternfly/react-core": "^4.6.1",
     "@patternfly/react-icons": "^4.1.1",
     "@patternfly/react-styles": "^4.1.1",
+    "axios": "^0.19.2",
```
Comment (gagansk) [inline package.json:85]: Resolve Merge conflict here.

File: src/app/Openshift/Openshift.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import { PageSection, Title }  from '@patternfly/react-core';
+import { Fetchdata } from '@app/fetchdata/fetchdata';
```
Comment (gagansk) [inline src/app/Openshift/Openshift.tsx:3]: Resolve merge conflict here.

**Review by gagansk — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts.

File: package.json (hunk 1 above)
Comment (shakyav) [reply package.json:85]: Conflict Resolved , axios is required to fetch data from an api so it needs to be added in package.json

File: src/app/Openshift/Openshift.tsx (hunk 2 above)
Comment (shakyav) [reply src/app/Openshift/Openshift.tsx:3]: Conflict resolved , import required to display the data on OpenShift tab, hence added .

**Review by gagansk — APPROVED** [review]
LGTM

**Pull request merged**